mkdocs/docs/HPC/running_batch_jobs.md

```diff
@@ -202,11 +202,10 @@ you will have to unload the packages one by one. When the
 $ module unload secondexample
 $ module list
 Currently Loaded Modulefiles: 
-Currently Loaded Modulefiles: 
-1) example/1.2.3                        5) impi/5.1.3.181-iccifort-2016.3.210-GCC-5.4.0-2.26 
-2) GCCcore/5.4.0                        6) imkl/11.3.3.210-iimpi-2016b 
-3) icc/2016.3.210-GCC-5.4.0-2.26        7) intel/2016b 
-4) ifort/2016.3.210-GCC-5.4.0-2.26      8) examplelib/1.2-intel-2016b
+1) example/1.2.3                        5) impi/5.1.3.181-iccifort-2023.3.210-GCC-5.4.0-2.26 
+2) GCCcore/5.4.0                        6) imkl/11.3.3.210-iimpi-2023a
+3) icc/2023.3.210-GCC-5.4.0-2.26        7) intel/2023a
+4) ifort/2023.3.210-GCC-5.4.0-2.26      8) examplelib/4.5.6-intel-2023a
```

Comment:
This is wrong, check output of `module show intel/2023a`:

* should be `GCCcore/12.3.0`
* no `icc/*`, only `intel-compilers/*`
* etc.

Reply:
in [use real module list in 'module load' and 'module unload' examples](https://github.com/hpcugent/vsc_user_docs/pull/710/commits/a3a953258d30097f742509882a19a3eed1455112), i used the `AMICA/2024.1.19-intel-2023a` module for `secondexample`. The dependencies should now be correct.